kong/client.go

```diff
@@ -363,3 +363,37 @@ func (c *Client) RootJSON(ctx context.Context) ([]byte, error) {
 
 	return body, nil
 }
+
+// SendConfig sends out the specified config to configured Admin API endpoint
+func (c *Client) SendConfig(ctx context.Context, config Configuration) error {
```

Comment:
Let's consider naming it `ReloadDeclarativeConfig` as this is how it's named in the official docs of the Admin API: https://docs.konghq.com/gateway/latest/admin-api/#reload-declarative-configuration. 

Probably a comment telling that it's valid only for the DB-less mode would be useful here.

Reply:
Done